account.go

```diff
@@ -308,10 +311,11 @@ func (account *Account) prepFunctionInvoke(ctx context.Context, messageType stri
 			return nil, err
 		}
 	}
-	s1, s2, err := account.Sign(txHash)
+	s1, s2, err := account.ks.Sign(account.sender, txHash)
```

Comment:
small thing - but this requires a context right?

Reply:
good catch, added